multicore_runtime/wsgi.py

```diff
@@ -43,6 +43,19 @@
 
 root_logger.setLevel(logging.INFO)
 
+# Enable Python Cloud Debugger if one is present. The debugger agent
+# communicates with the Cloud Debugger backend.
+#
+# The debugger has no effect on the application until snapshot is
+# requested.
+#
+# For details see: https://github.com/GoogleCloudPlatform/cloud-debug-python
+try:
+  import googleclouddebugger
+  googleclouddebugger.AttachDebugger()
+except BaseException:
```

Comment:
This is too broad. It will catch for instance SystemExit or MemoryError. Can we narrow it down to catch only the specific exceptions the debugger will raise, plus ImportError?